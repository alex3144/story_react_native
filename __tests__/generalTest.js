//connexion.js test
import 'react-native';
import React from 'react';
// import Profile from '../App/components/profile';
import Connexion from '../App/components/connexion';
// import Auth from '../App/actionAsync/auth/auth';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders correctly', () => {
   const tree = renderer.create(
    //  <Profile/>
     <Connexion/>
    //  <Auth />
   ).toJSON();
   expect(tree).toMatchSnapshot();
 });

