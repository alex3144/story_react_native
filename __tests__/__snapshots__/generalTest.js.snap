// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgb(54,54,54)",
      "flex": 1,
      "height": 1334,
      "padding": 30,
      "paddingBottom": 20,
      "paddingTop": 20,
      "width": 750,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    Profile
  </Text>
</View>
`;
